src/api/app/mailers/event_mailer.rb

```diff
@@ -58,6 +58,7 @@ def set_event_headers
     return unless @event
 
     headers['X-OBS-event-type'] = @event.template_name
+    headers['List-ID'] = "OBS Notification System <#{@event.template_name}.#{URI.parse(Configuration.obs_url).host.downcase}>}"
```

Comment:
we can always prepend I think, we just can't change the suffix anymore. I wasn't able to find a "repository" equivalent like GitHub does, because OBS requests (which are the bulk of my notifications) can have many source projects and potentially only one target project, but it's optional. 

do you want me to add a target project, if available? I think it would only be another prefix, so it could also be done later. wdyt?

i

Reply:
Yeah in general this belongs into `Event::Base.custom_headers` (and the subclasses) I think. It should give you a good hint on what's there to "identify" events.

Reply:
I'm not convinced that it should be an event specific custom header, but I'll explore that and update the PR when I have the time.